kod/object/active/holder/room/necarena.kod

```diff
@@ -619,14 +622,13 @@ messages:
       }
       else
       {
-         for iCntr in plGoToShrine
+         for each_obj in plGoToShrine
          {
-            Send(iCntr,@Teleport,#what=iCntr,#OutOfPlay=TRUE);
+            Send(self,@Teleport,#what=each_obj,#OutOfPlay=TRUE);
```

Comment:
nice find!

But what about just calling DelayedTeleport instead?  If the code isn't duplicated, there's no way the two versions can get out of sync.

Reply:
Yes this is a good suggestion.  I will implement and do a quick test and then update this.

Reply:
Ok - I implemented this suggestion and tested it just to be safe.  It works well and reads more clearly too.